src/nxdoc/nuxeo-server/indexing-and-query/configuring-the-elasticsearch-mapping.md

```diff
@@ -379,56 +381,6 @@ Using:
 SELECT * FROM Document WHERE /*+ES: INDEX(dc:title.ngram) ANALYZER(lowercase_analyzer) OPERATOR(match) */ dc:title = 'Foo'"));
 ```
 
-## Index the Main Attachment Content for Use with the Common Operator
-
-{{! multiexcerpt name='common-operator-main-attachment'}}{{> wistia_video id='nvdt4nzdq9'}}
```

Comment:
The multiexcerpt deleted here is used on the page Elasticsearch Hints Cheat Sheet, can I delete it there too?

Reply:
yes !